libs/models/src/lib/file.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose from 'mongoose';
+import { IFile } from '@kol-amelamdim/types';
+
+const FilesSchema = new mongoose.Schema(
+  {
+    key: {
+      type: String,
+      required: true,
+      unique: true,
+    },
```

Comment:
I think it's not necessary because we have automatic _id from mongoose

Reply:
Lets talk about it :)